### DISCUSSION TITLE
Support for Google Cloud Storage

### DISCUSSION THREAD

**Discussion by mycaule** [duckdb/duckdb#5068] [Ideas]
You can already import S3 Parquet or CSV files, but it would be great to support GCS like BigQuery does for external tables.

What's more, the product would also target GCP users.

**Comment by Alex-Monahan** [comment]
I believe this is already possible! Have a look at the examples here!
https://github.com/duckdb/duckdb/issues/1403

**Reply by mycaule** [reply]
Nice interoperability trick, it should be documented then.

**Reply by Alex-Monahan** [reply]
Good thought! If you are open to it, could you create a quick PR in our docs repo?
https://github.com/duckdb/duckdb-web

**Reply by adriens** [reply]
I have the same need since a few days :thought_balloon:
